### DISCUSSION TITLE
[OSPP-Week5] Debug: Continuous CRR and PLAS

### DISCUSSION THREAD

**Discussion by pilgrimygy** [JuliaReinforcementLearning/ReinforcementLearning.jl#434] [Archive]
@findmyway 
[Last week plan](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/discussions/409)

Week 5:
1. I will fix the bugs in Continuous CRR and PLAS.
2. I will discuss many designs in RL.jl with @findmyway.

**Comment by pilgrimygy** [comment]
I test the algorithms in Pendulum env, but the performance is poor. I don't know if there is a problem with my algorithms or with my dataset.

**Reply by Mobius1D** [reply]
Hey, you could use RLDatasets.jl for continuous control environments, d4rl would require mujoco but d4rl-pybullet does not need it.

If you have mujoco I would suggest you to use it so that you can be sure about whether the problem is with the dataset or the algorithm. 

The documentation is a bit sloppy, but if you read the docs in the source, the stuff would be pretty clear.

**Reply by Mobius1D** [reply]
It would return an iterator by default so that you would not need to worry about sampling. You can use it in a for loop directly.

**Reply by pilgrimygy** [reply]
Well, I am trying to do the experiments in d4rl. Thank you!

**Reply by Mobius1D** [reply]
You are welcome, lemme know if you have any suggestions.
